Code Samples/Fib/build.cmd

```diff
@@ -1,2 +1,2 @@
-SET PATH=%PATH%;%1
-g++ -g *.cpp -lpthread --std=c++11 -O0 -o %2
\ No newline at end of file
+@echo off
```

Comment:
Can you revert this build.cmd change? The modified one is not setting the path to the compiler.